fsharp.test/ParquetSharp.Test.FSharp.fsproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Whoops I didn't realise I had two `OutputType` entries. But I needed to add the `<OutputType>Library</OutputType>` line below for this to work on .NET Framework, otherwise I got errors like "Main module of program is empty"

Reply:
Ah yeah getting rid of both worked fine, thanks! :man_facepalming: